DEVELOPMENT.md

```diff
@@ -141,7 +166,8 @@ Then navigate to the Swagger: `http://localhost:8080/swagger-ui.html`
 - Check out [gng](https://github.com/gdubw/gng), it'll save you typing `./gradlew` over
   and over, and also takes care of knowing when you're not in the root directory so you
   don't have to figure out the appropriate number of `../`s.
-- In IntelliJ, instead of running the local server with `bootRun`, use the `TerraBatchAnalysisApplication` Spring
-  Boot configuration that IntelliJ auto-generates. To edit it, click on it (in the upper
+- In IntelliJ, instead of running the local server with `bootRun`, use the Spring
+  Boot configuration that IntelliJ auto-generates (`-cp cbas.service.main`, main class
+  `bio.terra.cbas.App`). To edit it, click on it (in the upper
```

Comment:
My intellij run mode has been broken for a little while and I'm not 100% sure why. I've just been using `../gradlew bootRun` but this encouraged me to try again. And... recreating the run mode with this configure just worked. Woohoo!